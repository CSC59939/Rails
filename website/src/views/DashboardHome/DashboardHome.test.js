import React from 'react';
import renderer from 'react-test-renderer';
import { shallow, mount, render, configure} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import { Button, Card, Icon } from 'antd';
import { DashboardHome } from '..';
import '../../utils/tests/test.css';
 configure({ adapter: new Adapter() });
 describe("DashboardHome Page", () => {
  it('Render DashboardHome page without any error', () => { // eslint-disable-line no-undef
    expect(renderer.create(<DashboardHome />).toJSON()).toMatchSnapshot();
  });
})
