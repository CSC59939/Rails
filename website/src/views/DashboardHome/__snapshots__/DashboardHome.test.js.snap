// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DashboardHome Page Render DashboardHome page without any error 1`] = `
<div
  className="DashboardHome"
>
  <div
    className="eventWrapper"
  >
    <span
      className="eventDate"
    >
      Oct. 24
    </span>
    <div
      className="events"
    >
      <div
        className="EventSummary"
        style={
          Object {
            "borderColor": "#d83a42",
            "color": "#d83a42",
          }
        }
      >
        <span
          className="course"
        >
          CSC 342
        </span>
        <span
          className="eventName"
        >
          Review of First Exam
        </span>
        <span
          className="dueDate"
        >
          3:30pm
        </span>
      </div>
      <div
        className="EventSummary"
        style={
          Object {
            "borderColor": "#62c4f9",
            "color": "#62c4f9",
          }
        }
      >
        <span
          className="course"
        >
          CSC 220
        </span>
        <span
          className="eventName"
        >
          Homework 1
        </span>
        <span
          className="dueDate"
        >
          5:00pm
        </span>
      </div>
    </div>
  </div>
  <div
    className="eventWrapper"
  >
    <span
      className="eventDate"
    >
      Oct. 25
    </span>
    <div
      className="events"
    >
      <div
        className="EventSummary"
        style={
          Object {
            "borderColor": "#d83a42",
            "color": "#d83a42",
          }
        }
      >
        <span
          className="course"
        >
          CSC 342
        </span>
        <span
          className="eventName"
        >
          First Exam
        </span>
        <span
          className="dueDate"
        >
          3:30pm
        </span>
      </div>
      <div
        className="EventSummary"
        style={
          Object {
            "borderColor": "#62c4f9",
            "color": "#62c4f9",
          }
        }
      >
        <span
          className="course"
        >
          CHEM 103
        </span>
        <span
          className="eventName"
        >
          Lab Report
        </span>
        <span
          className="dueDate"
        >
          7:00pm
        </span>
      </div>
    </div>
  </div>
  <div
    className="eventWrapper"
  >
    <span
      className="eventDate"
    >
      Oct. 30
    </span>
    <div
      className="events"
    >
      <div
        className="EventSummary"
        style={
          Object {
            "borderColor": "#d83a42",
            "color": "#d83a42",
          }
        }
      >
        <span
          className="course"
        >
          MATH 203
        </span>
        <span
          className="eventName"
        >
          Quiz
        </span>
        <span
          className="dueDate"
        >
          12:00pm
        </span>
      </div>
      <div
        className="EventSummary"
        style={
          Object {
            "borderColor": "#d83a42",
            "color": "#d83a42",
          }
        }
      >
        <span
          className="course"
        >
          CSC 301
        </span>
        <span
          className="eventName"
        >
          Midterm
        </span>
        <span
          className="dueDate"
        >
          6:30pm
        </span>
      </div>
      <div
        className="EventSummary"
        style={
          Object {
            "borderColor": "#62c4f9",
            "color": "#62c4f9",
          }
        }
      >
        <span
          className="course"
        >
          PHY 201
        </span>
        <span
          className="eventName"
        >
          Lab Report
        </span>
        <span
          className="dueDate"
        >
          11:59pm
        </span>
      </div>
    </div>
  </div>
  <div
    className="eventWrapper"
  >
    <span
      className="eventDate"
    >
      Oct. 31
    </span>
    <div
      className="events"
    >
      <div
        className="EventSummary"
        style={
          Object {
            "borderColor": "#62c4f9",
            "color": "#62c4f9",
          }
        }
      >
        <span
          className="course"
        >
          CSC 59929
        </span>
        <span
          className="eventName"
        >
          Outline of Research Paper
        </span>
        <span
          className="dueDate"
        >
          6:30pm
        </span>
      </div>
    </div>
  </div>
</div>
`;
